internal/channelz/types_linux.go

```diff
@@ -1,4 +1,4 @@
-// +build !appengine
```

Comment:
I believe this is redundant because of the filename: https://pkg.go.dev/go/build#hdr-Build_Constraints

Reply:
Done.